### PR TITLE
Add issue lifecycle tracker period jobs

### LINKED ISSUE #532: Automatically manage issue lifecycle
Let's use something like fejta-bot (it automatically add warnings on stale issues, and closes them if they're inactive for 90 days, among other stuff). A quick search in the GitHub repo didn't show me a straightforward way to set it up, so this might need a bit of digging.

Prow job config for fejta-bot:
https://github.com/kubernetes/test-infra/blob/4da53f46e256ccb3c2b06406cf8df9923907de02/config/jobs/kubernetes/test-infra/fejta-bot-periodics.yaml

### PR DESCRIPTION
**Pull request by Fredy-Z**
**Which issue(s) this PR fixes**:
Fixes #532

### CODE REVIEW

File: ci/prow/config.yaml [hunk 1]
```diff
@@ -5161,6 +5161,121 @@ periodics:
...
+- cron: "0 */12 * * *"
+  name: periodic-test-infra-stale
+  agent: kubernetes
+  decorate: true
+  extra_refs:
+  - org: knative
+    repo: test-infra
+  spec:
+    containers:
+    - image: gcr.io/k8s-prow/commenter:latest
```
Comment (chaodaiG) [inline ci/prow/config.yaml]: not sure if we should use latest

**Comment by chaodaiG** [comment]
@Fredy-Z , please delete [WIP] from the title if it's ready

**Comment by Fredy-Z** [comment]
@adrcunha I created a new bot for it - https://github.com/knative-issue-archive-robot, could you please make a new picture for it? Thanks :-)

**Comment by Fredy-Z** [comment]
It's done and the github token has been created on Prow cluster.

**Comment by chaodaiG** [comment]
/lgtm
/approve

/hold
please work with oncaller applying it

File: ci/prow/periodic_config.go [hunk 2]
```diff
@@ -327,3 +328,71 @@ func generateClearAlertsPeriodicJob() {
 	addVolumeToJob(&data.Base, "/secrets/cloudsql/monitoringdb", "monitoring-db-credentials", true, "")
 	executeJobTemplate("periodic clearalert", readTemplate(periodicCustomJob), "presubmits", "", data.PeriodicJobName, false, data)
 }
+
+func generateIssueTrackerPeriodicJobs() {
+	staleJobName := "periodic-test-infra-stale"
```
Comment (adrcunha) [inline ci/prow/periodic_config.go]: The "test-infra" in the job names make me think that these job only act on knative/test-infra, which is not true. Please use the same naming convention as the other generic jobs, like "ci-knative-issue-tracker-stale".

File: ci/prow/periodic_config.go [hunk 3]
```diff
@@ -327,3 +328,71 @@ func generateClearAlertsPeriodicJob() {
...
+func generateIssueTrackerPeriodicJob(jobName, labelFilter, updatedTime, comment string) {
+	var data periodicJobTemplateData
+	data.Base = newbaseProwJobTemplateData("knative/test-infra")
+	data.Base.Image = githubCommenterDockerImage
+	data.PeriodicJobName = jobName
+	data.CronString = issueTrackerPeriodicJobCron
+	data.Base.Command = "/app/robots/commenter/app.binary"
+
+	data.Base.Args = []string{
+		fmt.Sprintf(`|-
```
Comment (adrcunha) [inline ci/prow/periodic_config.go]: Why not `"--query=org:knative\n" + labelFiler`?

File: ci/prow/periodic_config.go [hunk 4]
```diff
@@ -327,3 +328,71 @@ func generateClearAlertsPeriodicJob() {
...
+        --query=org:knative
+        %s`, labelFilter),
+		fmt.Sprintf("--updated=%s", updatedTime),
```
Comment (adrcunha) [inline ci/prow/periodic_config.go]: Why not `"--updated=" + updatedTime` ?

File: ci/prow/periodic_config.go (hunk 2 above)
Comment (Fredy-Z) [reply ci/prow/periodic_config.go]: Done

File: ci/prow/periodic_config.go (hunk 3 above)
Comment (Fredy-Z) [reply ci/prow/periodic_config.go]: Done.

File: ci/prow/periodic_config.go (hunk 4 above)
Comment (Fredy-Z) [reply ci/prow/periodic_config.go]: Done.

File: ci/prow/periodic_config.go [hunk 5]
```diff
@@ -383,10 +383,10 @@ func generateIssueTrackerPeriodicJob(jobName, labelFilter, updatedTime, comment
 	data.Base.Command = "/app/robots/commenter/app.binary"
 
 	data.Base.Args = []string{
-		fmt.Sprintf(`|-
+		`|-
```
Comment (adrcunha) [inline ci/prow/periodic_config.go]: Funny, can't you just use \n instead of the multi-line string for a single line?

Comment (Fredy-Z) [reply ci/prow/periodic_config.go]: I tried but the indentation was mysteriously incorrect :(

**Pull request merged**